client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -544,10 +544,16 @@ export default {
     setInitialValues () {
       this.values = { ...this.initValues }
       // Set default values to ensure reactivity.
-      if (typeof this.values.submitter === 'undefined' || Object.keys(this.values.submitter).length === 0) {
+      if (typeof this.values.submitter !== 'undefined' && typeof this.values.submitter.institution === 'undefined') {
```

Comment:
```suggestion
      if (hasOwnProp(this.values), 'submitter') && !hasOwnProp(this.values.submitter, 'institution') {
```
(hasOwnProp has to be importet eventually)